packages/css/avatar.css

```diff
@@ -28,6 +28,16 @@
   }
 }
 
+.ds-avatar--icon::before {
```

Comment:
Kan denne heller være `.ds-avatar:empty::before`, så slipper man å ha noe JS for å legge til `--icon`? ☺️